src/repr/scalar/mod.rs

```diff
@@ -745,6 +745,46 @@ impl Default for Interval {
     }
 }
 
+impl std::ops::Add for Interval {
+    type Output = Self;
+    // We need subtraction and boolean operators to sensibly
+    // add intervals to one another.
+    #[allow(clippy::suspicious_arithmetic_impl)]
```

Comment:
I would _slightly_ prefer this to say: `Since durations can only be positive, we need subtraction and boolean operators inside the Add impl`